pkg/terraform/exec/plugins/libvirt.go

```diff
@@ -4,7 +4,7 @@ package plugins
 
 import (
 	"github.com/dmacvicar/terraform-provider-libvirt/libvirt"
-	"github.com/hashicorp/terraform-plugin-sdk/plugin"
+	"github.com/hashicorp/terraform/plugin"
```

Comment:
this seems opposite of other changes?

Reply:
libvirt provider is not yet compatible with the plugin-sdk